prow/cmd/deck/main.go

```diff
@@ -33,21 +34,25 @@ import (
 
 	"github.com/NYTimes/gziphandler"
 	"github.com/ghodss/yaml"
+	"github.com/gorilla/sessions"
 	"github.com/sirupsen/logrus"
 
 	"k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/gitoauth"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/pjutil"
 	"k8s.io/test-infra/prow/pluginhelp"
 	"k8s.io/test-infra/prow/userdashboard"
 )
 
 var (
-	configPath   = flag.String("config-path", "/etc/config/config", "Path to config.yaml.")
-	buildCluster = flag.String("build-cluster", "", "Path to file containing a YAML-marshalled kube.Cluster object. If empty, uses the local cluster.")
-	tideURL      = flag.String("tide-url", "", "Path to tide. If empty, do not serve tide data.")
-	hookURL      = flag.String("hook-url", "", "Path to hook plugin help endpoint.")
-	userUrl      = flag.String("user-url", "", "Path to hook user dashboard endpoint.")
+	configPath       = flag.String("config-path", "/etc/config/config", "Path to config.yaml.")
+	buildCluster     = flag.String("build-cluster", "", "Path to file containing a YAML-marshalled kube.Cluster object. If empty, uses the local cluster.")
+	tideURL          = flag.String("tide-url", "", "Path to tide. If empty, do not serve tide data.")
+	hookURL          = flag.String("hook-url", "", "Path to hook plugin help endpoint.")
+	userUrl          = flag.String("user-url", "", "Path to hook user dashboard endpoint.")
+	gitAppClientFile = flag.String("gitapp-client-file", "/ect/github/app", "Path to the file containing the Git App Client secret.")
+	cookieSecretFile = flag.String("cookie-secret", "/ect/github/app", "Path to the file containing the cookie secret key.")
```

Comment:
should probably be "/etc/cookie-secret" instead of "/ect/github/app"

Reply:
done